pkg/lint/rules/values.go

```diff
@@ -48,6 +49,18 @@ func validateValuesFileExistence(valuesPath string) error {
 }
 
 func validateValuesFile(valuesPath string) error {
-	_, err := chartutil.ReadValuesFile(valuesPath)
-	return errors.Wrap(err, "unable to parse YAML")
+	values, err := chartutil.ReadValuesFile(valuesPath)
+	if err != nil {
+		return errors.Wrap(err, "unable to parse YAML")
+	}
+
+	schemaPath := strings.Replace(valuesPath, ".yaml", ".schema.yaml", 1)
```

Comment:
What if valuesPath was `values.yaml.yaml`? It's a crazy edge case, but this would only replace the first instance of `.yaml` rather than the file extension.

You could use `filepath.Ext` to find the file extension, then replace it with .schema.yaml.

```
ext := filepath.Ext(valuesPath)
schemaPath = valuesPath[:len(valuesPath)-len(ext)] + ".schema.yaml"
```